### PR TITLE
197 clock with rounded corners

### PR DESCRIPTION
**Pull request by mastermichi**
-  show clock layout with round corners

### CODE REVIEW

File: src/com/firebirdberlin/nightdream/ui/ClockLayout.java [hunk 1]
```diff
@@ -271,7 +273,10 @@ private void toggleDivider() {
             if (showDivider) {
                 divider.setVisibility(VISIBLE);
             }
-            setBackgroundColor(Color.parseColor("#44000000"));
+            GradientDrawable shape =  new GradientDrawable();
+            shape.setCornerRadius( Settings.getRoundCorners(context) );
```
Comment (firebirdberlin) [inline src/com/firebirdberlin/nightdream/ui/ClockLayout.java:277]: My suggestion is to remove the preference and to set the radius to 50
```suggestion
            shape.setCornerRadius(50);
```

File: src/com/firebirdberlin/nightdream/ui/ClockLayout.java [hunk 2]
```diff
@@ -19,6 +20,7 @@
 import android.view.View;
 import android.widget.LinearLayout;
 import android.widget.TextView;
+import android.widget.Toast;
```
Comment (firebirdberlin) [inline src/com/firebirdberlin/nightdream/ui/ClockLayout.java:23]: This is line not in use and can be removed.

File: src/com/firebirdberlin/nightdream/Settings.java [hunk 3]
```diff
@@ -38,6 +38,7 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Locale;
+import java.util.Map;
```
Comment (firebirdberlin) [inline src/com/firebirdberlin/nightdream/Settings.java:41]:
```suggestion
```

File: res/xml-v14/preferences_appearance.xml [hunk 4]
```diff
@@ -50,6 +50,7 @@ ToDo
             app:key="fadeClock"
             app:title="@string/fade_clock" />
 -->
+
```
Comment (firebirdberlin) [inline res/xml-v14/preferences_appearance.xml:53]:
```suggestion
```

File: src/com/firebirdberlin/nightdream/Settings.java (hunk 3 above)
Comment (mastermichi) [reply src/com/firebirdberlin/nightdream/Settings.java:41]: done

File: src/com/firebirdberlin/nightdream/ui/ClockLayout.java (hunk 2 above)
Comment (mastermichi) [reply src/com/firebirdberlin/nightdream/ui/ClockLayout.java:23]: done

File: res/xml-v14/preferences_appearance.xml (hunk 4 above)
Comment (mastermichi) [reply res/xml-v14/preferences_appearance.xml:53]: from master for future release

**Pull request closed without merging**